scripts/quests/bastok/A_Question_of_Faith.lua

```diff
@@ -84,18 +84,29 @@ quest.sections =
             ['Rakorok'] =
             {
                 onTrigger = function(player, npc)
-                    if player:hasKeyItem(xi.ki.DAWN_TALISMAN) and quest:getVar(player, 'Prog') == 0 then
+                    if
+                        player:hasKeyItem(xi.ki.DAWN_TALISMAN) and
+                        quest:getVar(player, 'Prog') == 0
+                    then
                         npc:messageText(npc, 7733)
```

Comment:
so this whole block needs changing quest frame work needs to return something to override lua and default actions:
```lua
if npcUtil.popFromQM(player, npc, 16822456, {claim = true, hide = 0}) then
    quest:messageText(npc, ID.text.ENUM_REPLACEMENT_1)
    retrun quest:messageText(npc, ID.text.ENUM_REPLACEMENT_2)
```

Reply:
and not use text ids for messages, make an enum for it in the id lua for its zone

Reply:
Thank you for catching all these and teaching! :)